TRANSCRIPTION.md

```diff
@@ -4,9 +4,149 @@
 ## Segments
 
 | Segment | Occurrence | BIPA | CLTS SoundClass |
-||
-
-(0 rows)
+|:----------|-------------:|:-------|:------------------|
+| a | 188 | ✓ | ✓ |
+| u | 164 | ✓ | ✓ |
+| i | 161 | ✓ | ✓ |
+| ŋ | 147 | ✓ | ✓ |
+| p | 138 | ✓ | ✓ |
+| n | 114 | ✓ | ✓ |
+| e | 113 | ✓ | ✓ |
+| t | 106 | ✓ | ✓ |
+| ³³ | 104 | ✓ | ✓ |
+| j | 102 | ✓ | ✓ |
+| + | 98 | ✓ | ✓ |
+| ³⁵ | 98 | ✓ | ✓ |
+| k | 89 | ✓ | ✓ |
+| ⁴⁴ | 80 | ✓ | ✓ |
+| ɔ | 74 | ✓ | ✓ |
+| o | 71 | ✓ | ✓ |
+| ³¹ | 71 | ✓ | ✓ |
+| m | 62 | ✓ | ✓ |
+| w | 50 | ✓ | ✓ |
+| ²² | 50 | ✓ | ✓ |
+| l | 49 | ✓ | ✓ |
+| ⁵³ | 47 | ✓ | ✓ |
+| ə | 38 | ✓ | ✓ |
+| ɛ | 37 | ✓ | ✓ |
+| ⁴² | 37 | ✓ | ✓ |
+| ⁵⁵ | 36 | ✓ | ✓ |
+| ²⁴ | 32 | ✓ | ✓ |
+| ȵ | 29 | ✓ | ✓ |
+| s | 28 | ✓ | ✓ |
+| b | 27 | ✓ | ✓ |
+| d | 27 | ✓ | ✓ |
+| h | 25 | ✓ | ✓ |
+| tɕ | 24 | ✓ | ✓ |
+| ⁴³ | 21 | ✓ | ✓ |
+| ³² | 20 | ✓ | ✓ |
+| v | 19 | ✓ | ✓ |
+| ¹ | 18 | ✓ | ✓ |
+| aː | 17 | ✓ | ✓ |
+| ts | 17 | ✓ | ✓ |
+| ɦ | 17 | ✓ | ✓ |
+| ʔ | 16 | ✓ | ✓ |
+| X/¹⁵ | 15 | ✓ | ✓ |
+| tʰ | 14 | ✓ | ✓ |
+| æ | 14 | ✓ | ✓ |
+| ɕ | 14 | ✓ | ✓ |
+| q | 13 | ✓ | ✓ |
+| z | 13 | ✓ | ✓ |
+| ²¹ | 13 | ✓ | ✓ |
+| ³ | 12 | ✓ | ✓ |
+| ¹³ | 12 | ✓ | ✓ |
+| ¹¹ | 12 | ✓ | ✓ |
+| kʰ | 11 | ✓ | ✓ |
+| r | 11 | ✓ | ✓ |
+| tsʰ | 11 | ✓ | ✓ |
+| ɑ | 11 | ✓ | ✓ |
+| H/⁵¹ | 10 | ✓ | ✓ |
+| ʈ | 10 | ✓ | ✓ |
+| θ | 10 | ✓ | ✓ |
+| ⁵⁴⁵ | 9 | ✓ | ✓ |
+| g | 8 | ✓ | ✓ |
+| ²²¹ | 8 | ✓ | ✓ |
+| ɤ | 8 | ✓ | ✓ |
+| ɯ | 8 | ✓ | ✓ |
+| ⁵ | 8 | ✓ | ✓ |
+| tʃ | 7 | ✓ | ✓ |
+| u̯ | 7 | ✓ | ✓ |
+| ȵ̥ | 7 | ✓ | ✓ |
+| ɣ | 7 | ✓ | ✓ |
+| ⁴¹ | 7 | ✓ | ✓ |
+| ⁶/⁵¹ | 7 | ✓ | ✓ |
+| ⁿb | 7 | ✓ | ✓ |
+| f | 6 | ✓ | ✓ |
+| ¹² | 6 | ✓ | ✓ |
+| ȶ | 6 | ✓ | ✓ |
+| ɪ | 6 | ✓ | ✓ |
+| ʐ | 6 | ✓ | ✓ |
+| ʑ | 6 | ✓ | ✓ |
+| c | 5 | ✓ | ✓ |
+| l̥ | 5 | ✓ | ✓ |
+| ð | 5 | ✓ | ✓ |
+| ɐ | 5 | ✓ | ✓ |
+| ɔ̃ | 5 | ✓ | ✓ |
+| ɬ | 5 | ✓ | ✓ |
+| ʉ | 5 | ✓ | ✓ |
+| ⁴ | 5 | ✓ | ✓ |
+| ⁷/³³ | 5 | ✓ | ✓ |
+| i̯ | 4 | ✓ | ✓ |
+| sʰ | 4 | ✓ | ✓ |
+| tɕʰ | 4 | ✓ | ✓ |
+| ² | 4 | ✓ | ✓ |
+| ²¹³ | 4 | ✓ | ✓ |
+| ɭ | 4 | ✓ | ✓ |
+| ʃ | 4 | ✓ | ✓ |
+| θʰ | 4 | ✓ | ✓ |
+| ⁸/⁴⁴ | 4 | ✓ | ✓ |
+| mʐ | 3 | ✓ | ✓ |
+| ²³² | 3 | ✓ | ✓ |
+| õ | 3 | ✓ | ✓ |
+| ɕʰ | 3 | ✓ | ✓ |
+| ɖʐ | 3 | ✓ | ✓ |
+| ɳ | 3 | ✓ | ✓ |
+| ˀw | 3 | ✓ | ✓ |
+| ⁵⁴ | 3 | ✓ | ✓ |
+| kʷ | 2 | ✓ | ✓ |
+| l̥ʰ | 2 | ✓ | ✓ |
+| oː | 2 | ✓ | ✓ |
+| pʐ | 2 | ✓ | ✓ |
+| tʂ | 2 | ✓ | ✓ |
```

Comment:
tʂ is not correct in terms of BIPA, please paste it [here](http://digling.org/calc/clts/) to get the right version.

Reply:
pʐ should be `p ʐ`